base/dotenv.default.mustache

```diff
@@ -0,0 +1,3 @@
+PORT=
+RETHINK_HOST=
+RETHINK_NAME=
```

Comment:
We use example.env rather than .env because .env should never evar evar be committed to source control. The danger is that you put some testing specific stuff in .env. You accidentally commit that change to git. That file then wends its way through to staging and/or prod with the testing keys/config still in there and Bad Things Happen.

So, we create example.env to protect against that eventuality, but still make it simple for people to just `cp example.env .env` and get on with writing some code.

I don't think we need to worry about people deleting example.env since that would be a silly thing to do. It would mean that any future travellers to the project would have no idea what was expected to be in the .env file.

tl:dr, let's ditch dotenv.default and just use the example one.